erpnext/manufacturing/doctype/work_order/work_order.js

```diff
@@ -347,6 +347,21 @@ frappe.ui.form.on("Work Order", {
 		});
 	},
 
+	bom_barcode: function(frm) {
+		frm.call({
+			doc: frm.doc,
+			method: "get_item_bom_from_barcode",
+			args: {barcode: frm.doc.bom_barcode},	
+		}).then((r) => {
+			if (r.message){
+				frm.set_value("production_item","");
```

Comment:
```suggestion
				frm.set_value("production_item", "");
```
add single space after comma everywhere